package.json

```diff
@@ -22,6 +22,7 @@
     "clean": "rm -rf package-lock.json && rm -rf node_modules && rm -rf coverage && rm -rf .nyc_output",
     "healthcheck": "node ./bin/healthcheck",
     "lint": "eslint lib/ bin/iota-manager test/ --cache --fix",
+    "start": "node ./bin/iotagent-manager",
```

Comment:
I think npm targets are described here in this repo: https://github.com/telefonicaid/iotagent-manager#-development-documentation

Reply:
Fixed 4ba0df9